recipe/meta.yaml

```diff
@@ -1,15 +1,15 @@
 {% set name = "pydocstyle" %}
-{% set version = "2.0.0" %}
+{% set version = "2.1.1" %}
 {% set hash_type = "sha256" %}
-{% set hash = "49680b7ea8c18e911135167e9f6986a3829031b55f2c737e6d2c398df38aafb5" %}
+{% set hash = "4d5bcde961107873bae621f3d580c3e35a426d3687ffc6f8fb356f6628da5a97" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   fn: {{ name }}-{{ version }}.zip
```

Comment:
Need `.tar.gz` here too or this can just be dropped.

Reply:
Thanks, done.